src/components/BreadcrumbMenu/BreadcrumbMenu.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import styled from 'styled-components';
+import { NavLink, useParams, matchPath } from 'react-router-dom';
+import { HiMenuAlt3 } from 'react-icons/hi';
+import useBreadcrumbs from 'use-react-router-breadcrumbs';
+
+const DynamicBreadcrumb = () => {
+  const { id } = useParams();
+  return <span>{id}</span>;
+};
+
+const routes = [
+  { path: '/', breadcrumb: 'Home' },
+  { path: '/admin', breadcrumb: null },
+  { path: '/admin/cohorts/:id', breadcrumb: DynamicBreadcrumb },
+];
+
+const excludePaths = ['/', '/admin/cohorts/:id'];
+
+const isRouteExcluded = (pathname: string) => {
+  return excludePaths.find((path) => matchPath({ path }, pathname));
+};
+
+const navLinkStyles = {
+  color: 'white',
+};
+
+const activeStyle = {
+  fontWeight: 'bold',
+  fontSize: '1.125em',
```

Comment:
hmm so the fontsize is different for when it's active? wouldn't it look inconsistent if the font size is different? Maybe this should be in the general `navLinkStyles` instead?

Reply:
My thinking was to make the current breadcrumb more obvious by making it just a little bigger. I can remove it, though!